tests/drum/test_drum_push.py

```diff
@@ -40,17 +49,12 @@ def test_drum_push_training(
         problem,
         language,
         tmp_path,
-        sklearn_drop_in_env,
     ):
         custom_model_dir = _create_custom_model_dir(
-            resources,
-            tmp_path,
-            framework,
-            problem,
-            language,
+            resources, tmp_path, framework, problem, language, is_training=True
         )
 
-        env_id, _ = sklearn_drop_in_env
+        env_id = "5e8c889607389fe0f466c72d"
```

Comment:
Why is this done?

Reply:
@yakov-g requested that I move this out of the functional tests, so this was to avoid moving the sklearn_drop_in_env in the functional conftest up a level....but maybe there's no reason not to do that?

Reply:
if this is hitting DR it should be inside of functional tests tho

Reply:
calling push from the command line will always hit DR, correct? in which case @yakov-g I think Josh is right that this does belong in functional tests after all....I can open a new PR to just take care of the unsupervised part of this